### PR TITLE
New metrics to observe the content of Thoth Graph database

### PR DESCRIPTION
**Pull request by pacospace**
Depends-On: https://github.com/thoth-station/storages/pull/579

### CODE REVIEW

File: thoth/metrics_exporter/jobs.py [hunk 1]
```diff
@@ -48,8 +48,7 @@ def get_retrieve_unsolved_pypi_packages():
     asyncio.set_event_loop(loop)
 
     try:
-        # janusgraph is a hostname injected into the pod by the 'janusgraph' service object
-        graph = GraphDatabase()
+        graph = GraphDatabase.create('janusgraph.test.thoth-station.ninja')
```
Comment (fridex) [inline thoth/metrics_exporter/jobs.py]: Please use just:

```
 graph = GraphDatabase()
```

to pick graphdatabase based on configuration in env.

File: thoth/metrics_exporter/jobs.py [hunk 2]
```diff
@@ -119,15 +118,29 @@ def get_solver_documents(solver_name: str = None):
...
+
+        # Non-Optimized query
+        number_solver_documents = asyncio.get_event_loop().run_until_complete(graph_db.
+        g.E()
+            .has("__label__", "solved")
+            .valueMap()
+            .select("solver_document_id")
+            .dedup()
+            .count()
+            .next())
```
Comment (fridex) [inline thoth/metrics_exporter/jobs.py]: Feel free to directly supply this to thoth-storages not to database-specific queries split in the project.

File: thoth/metrics_exporter/jobs.py [hunk 3]
```diff
@@ -136,23 +149,147 @@ def get_analyzer_documents():
     loop = asyncio.new_event_loop()
     asyncio.set_event_loop(loop)
 
-    graph_db = GraphDatabase.create()
+    graph_db = GraphDatabase.create('janusgraph.test.thoth-station.ninja')
```
Comment (goern) [inline thoth/metrics_exporter/jobs.py]: see firdos comment above

**Comment by goern** [comment]
regate

**Pull request merged**